addons/arsenal/functions/fnc_sortStatement_mod.sqf

```diff
@@ -0,0 +1,24 @@
+#include "script_component.hpp"
+/*
+ * Author: SynixeBrett
+ * Statement for the mod sort.
+ *
+ * Arguments:
+ * 0: item config path (CONFIG)
+ *
+ * Return Value:
+ * Mod name to sort by
+ *
+ * Public: No
+*/
+
+private _dlc = "";
+private _addons = configSourceAddonList _this;
```

Comment:
Other sort functions have `params ["_config"]` instead of `_this` directly. Change this for consistency.